### PR TITLE
Clear CoreData when user logs out

### PR DESCRIPTION
**Pull request by bjrne**
This adds methods to the CoreDataHelper to remove entities from CoreData.

### CODE REVIEW

**Comment by invliD** [comment]
Is there no other way to do this?

**Comment by bjrne** [comment]
If you find a better one, im open for it. But something like this is what Apple showed at WWDC'15: https://developer.apple.com/videos/play/wwdc2015/220/ (starting at ~14:10)

File: xikolo-common/Data/Helper/CoreDataHelper.swift [hunk 1]
```diff
@@ -67,4 +67,26 @@ class CoreDataHelper {
...
 
+    static func clearCoreDataStorage() {
+        clearCoreDataEntity("Course")
+        clearCoreDataEntity("CourseSection")
+        clearCoreDataEntity("CourseItem")
+        clearCoreDataEntity("Content")
+        clearCoreDataEntity("Video")
+        clearCoreDataEntity("RichText")
+        clearCoreDataEntity("Quiz")
+        clearCoreDataEntity("NewsArticle")
```
Comment (invliD) [inline xikolo-common/Data/Helper/CoreDataHelper.swift]: What about `managedObjectContext.entitiesByName` and iterating the dict's keys? That way we won't be able to forget adding entities here.

Comment (bjrne) [inline xikolo-common/Data/Helper/CoreDataHelper.swift]: I didn't know about that, I thought you meant the general approach. I'll add that later.

Comment (invliD) [inline xikolo-common/Data/Helper/CoreDataHelper.swift]: I did. But after some research I couldn't find a better method. Only this improvement.

**Pull request merged**